python-threatexchange/threatexchange/cli/main.py

```diff
@@ -273,9 +273,9 @@ def _setup_logging(level_str: str, *, initial: bool = False) -> None:
 
 def inner_main(
     args: t.Optional[t.Sequence[t.Text]] = None,
-    state_dir: pathlib.Path = pathlib.Path("~/.threatexchange"),
```

Comment:
Why this change?

Reply:
This was a bad move. I thought it would be cleaner to move into the method body because it appeared unused. I hadn't considered the tests.

Reply:
This seems reasonable, looks like the format/lint `black` is complaining though